frontend/src/pages/ContributionView.jsx

```
@@ -0,0 +1,106 @@
+import { useParams } from 'react-router-dom';
+
+function ContributionView() {
+  const { id } = useParams();
+
+  return (
+    <div className="min-h-screen bg-white">
+      <Navbar />
+      
+      {/* Breadcrumb Navigation */}
+      <div className="flex items-center px-24 py-2 mt-[60px] text-sm text-purple-600 bg-gray-50">
+        <button className="hover:bg-gray-100 px-3 py-1 rounded-md">Hide menu</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)